index.js

```diff
@@ -0,0 +1,151 @@
+// task__1
+
+const sayHello = function (name) {
+    if (name == 'Mark') {
+        return console.log(`Hi, ${name}`);
```

Comment:
you have to `return` value, not `console.log`. Fix it here and in the code below